src/components/Badge/Badge.driver.ts

```diff
@@ -25,5 +26,6 @@ export const badgeDriverFactory = (base: UniDriver): BadgeDriver => {
     isPrimary: () => hasPriority(base, BADGE_PRIORITY.primary),
     isLight: () => hasPriority(base, BADGE_PRIORITY.light),
     isDefault: () => hasPriority(base, BADGE_PRIORITY.default),
+    getIcon: () => base.attr('.icon'),
```

Comment:
Using the class name `.icon` won't necessary work, since it depends on the hashing that's used on the compiled bundle.
So instead of classes please use a `data-` hook (like here: https://github.com/wix/wix-ui-tpa/pull/376/files#diff-da702d7b76d4fd862bd1a8e985077d14R12)